content/coronavirus_landing_page.yml

```diff
@@ -226,8 +226,7 @@ content:
       url: https://www.youtube.com/user/Number10gov/videos
       previous_videos_text: View past coronavirus press conferences on YouTube
       next_conference_text: The next live press conference will be shown here
-    date: 19th April 2020
-    show_video: false
+    time:
   live_stream_enabled: false
```

Comment:
I wonder, in the absence of docs, whether we should add a comment here telling people to enable/disable the live stream via https://collections-publisher.publishing.service.gov.uk/coronavirus ?

Reply:
I'm about to update the docs, but will leave a comment too good idea.